src/content/landing.ts

```diff
@@ -217,6 +203,7 @@ export const getLanding = (t: TFunction): CustomPageContent => ({
                     title: 'Remix',
                     icon: remixLogo,
                     commands: [
+                        '# probably you do not need it, since CRA is deprecated https://github.com/facebook/create-react-app?tab=readme-ov-file#deprecated',
```

Comment:
Should this comment be in the Remix section? Or CRA?)

Reply:
fixed, thanks https://github.com/gravity-ui/landing/pull/357/commits/572739d079a317e2aa71d0e4573dfaa52388d1c1